recipes/multi-hauser/Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM alpine:latest
+RUN apk update && apk add --no-cache \
+  supervisor \
+  curl \
+  gettext
+
+ARG HAUSER_VERSION=1.0.0
```

Comment:
To future proof and support newer releases without updating the recipe, you could also get the latest download URL with this `HAUSER_DOWNLOAD=$(curl -s https://api.github.com/repos/fullstorydev/hauser/releases/latest | grep "browser_download_url.*linux_x86_64.tar.gz" | cut -d : -f 2,3 | tr -d \")`.  Though if we want to keep it simple, what we have is pretty clear.

Reply:
Yeah, I saw that solution too, but went with this option since it was deterministic. If you feel strongly, I can update it   to be latest.

Reply:
Nah, just wanted to float it.  Seems like one more thing that will eventually break, and it's far more clear how to simply update the version value.